client/packages/programs/src/JsonForms/common/components/Autocomplete.tsx

```diff
@@ -102,12 +110,12 @@ const UIComponent = (props: ControlProps) => {
             flexBasis: '100%',
           }}
           options={options}
-          value={localData ?? { value: '', label: '' }}
+          value={localData ?? null}
```

Comment:
null instead of empty object here means the clear `X` won't show when the value is empty

Reply:
Should localData be nullable, or undefined ?

Reply:
Aw noice

Reply:
[f41caad](https://github.com/msupply-foundation/open-msupply/pull/6678/commits/f41caad5e06319e23f360c7f11fdc0a40db19638)